src/node_crypto.cc

```diff
@@ -3994,6 +3994,19 @@ bool SignBase::GetRSAOptions(Environment* env, v8::Local<v8::Object> options,
   return true;
 }
 
+static bool ApplyRSAOptions(EVP_PKEY* pkey, EVP_PKEY_CTX* pkctx, int padding, int salt_len) {
```

Comment:
Already fixed in the last commit, sorry. The problem is that I cannot perform linting on Windows and cannot run the tests on Linux, so I have to push from my Windows machine after running the tests before linting it on my Linux machine.